.github/workflows/integration-test-src.yml

```diff
@@ -89,6 +89,23 @@ jobs:
           comment_mode: "create new"
           check_name: "SRC java-sdk-cluster-memory-0 Report"
           comment_title: "SRC java-sdk-cluster-memory-0 Report"
+      - name: tar test report
+        if: ${{ failure() }}
+        run: tar -zcvf allure-results.tar.gz test/integration-test/openmldb-test-java/openmldb-sdk-test/target/allure-results
+      - name: Send Email
```

Comment:
GItHub will send fail email, config via https://github.com/settings/notifications ?

Reply:
This can't send the desired attachment, and can't send the failed mail according to the job